### ISSUE TITLE
Support custom buttons in modal-footer?

### ISSUE THREAD

**Issue by Andreas-Hjortland** [PWKad/aurelia-bs-modal#30]
Would it be possible to add support for custom buttons in the `<modal-footer>` element? As of now you only support `<button>` and `<au-button>` which doesn't allow custom elements.

In my case I use a custom element `<ladda>` which gives the submit button a spinner when you click it.

**Comment by plwalters** [comment]
@Andreas-Hjortland Yes this is fine we can certainly switch to a more open content selector, PR accepted.

**Comment by atsu85** [comment]
to resolve this issue, please merge my pull request https://github.com/PWKad/aurelia-bs-modal/pull/33  and run `gulp build` to update dist folder
